src/package/packageConvert.ts

```diff
@@ -238,8 +240,10 @@ export async function createPackageVersionCreateRequest(
   await fs.promises.mkdir(packageVersBlobDirectory, { recursive: true });
   await fs.promises.mkdir(packageVersMetadataFolder, { recursive: true });
 
+  // Use CLI seedMetadata if provided, otherwise fall back to project configuration
```

Comment:
nit: CLI commands aren't the only way to call this.  There are customers who use this library directly and in the future there will be MCP tools call it.